parcel/ParcelTendencies.jl

```diff
@@ -190,9 +190,10 @@ end
 function condensation(params::CondParams, PSD, state, ρ_air)
     FT = eltype(state)
     (; aps, tps) = params
-    (; Sₗ, T, Nₗ) = state
+    (; Sₗ, T, Nₗ, qₗ) = state
     Gₗ = CMO.G_func(aps, tps, T, TD.Liquid())
-    return 4 * FT(π) / ρ_air * (Sₗ - 1) * Gₗ * PSD.rₗ * Nₗ
+    return Nₗ > FT(0) && qₗ > FT(0) ?
```

Comment:
Is this check needed? Maybe it would be better to ensure that `PSD.rl` is zero when q and N are zero? I think we are actually already doing it for `N` but maybe not for `q` for all the supported distributions...

This way it would also work for the deposition growth